noir-projects/noir-protocol-circuits/crates/rollup-lib/src/root/mod.nr

```diff
@@ -5,68 +5,19 @@ mod root_rollup_public_inputs;
 use root_rollup_inputs::RootRollupInputs;
 use root_rollup_public_inputs::RootRollupPublicInputs;
 
-// TODO: Move all the following code to different files
-use dep::types::{constants::NUMBER_OF_L1_L2_MESSAGES_PER_ROLLUP, utils::uint256::U256, hash::sha256_to_field};
-
-// See `test_message_input_flattened_length` on keeping this in sync,
-// why its here and how this constant was computed.
-global NUMBER_OF_L1_L2_MESSAGES_PER_ROLLUP_NUM_BYTES: u32 = 512;
-
-// Computes the messages hash from the leaves array
-//
-// TODO(Miranda): remove? This appears to be unused
-// Returns the hash truncated to one field element
```

Comment:
Removed this code as it's unused as has been since I added this comment (March?)